lib/exqlite/connection.ex

```diff
@@ -230,8 +230,8 @@ defmodule Exqlite.Connection do
         end
 
       mode
-      when mode in [:deferred, :immediate, :exclusive, :transaction] and
-             transaction_status == :transaction ->
+      when mode in [:deferred, :immediate, :exclusive, :transaction] -> #and
+             #transaction_status == :transaction ->
```

Comment:
Let's go ahead and delete the commented code. We can always revert it back if need be.